Magi/Rojyar of the Nizari/Rojyar, the Mustajib.md

```diff
@@ -0,0 +1,126 @@
+# Rojyar, the Mustajib
+
+Rojyar is now a powerful member of the Nizari summoners but, before becoming a mustajib, he was initiated into the summoning arts bythe Mulhidun to be able to control demons and spirits. These initiations bound his magic to specific demons and one of them become really close to Royjar, she call herself Yassin. Royjar is a skilled poisoner, he can concoct poisons and then increase their potency using Physic, he is also able to change the humors in someone’s body to make them react in a predictable way. His goal is to serve the Nizari, while slowly but eventually becoming one of the most powerful *imam-qa’im*.
+
+**Characteristics:** Int +2, Per +0, Pre +2, Com +2, Str -2, Sta +0, Dex +1, Qik +0  
+**Size:** 0  
+**Age:** 25  
+**Decrepitude:** 0 (0)  
+**Warping Score:** 2 (25)  
+**Confidence Score:** 2 (6)*
+
+\* A Vituperation caused the Confidence Score to increase
+
+## Virtues and Flaws:
+
+- Mustajib
+- Physics
+- Summoning
+- Commanding
+- Convoluted Mind
+- Dark Magic
+- Fida'i
+- Puissant Penetration
+- Greater Magical Defences
+- Second Sight
+- Trained Assassin (x2)
+- Skilled Mentor
+- Bound Spells
+- Infernal Spirit Companion
+- Disfigured
+- Divine Stigmatic
+- Dutybound
+- Obsessed
+
+**Personality Trait:** Calm +2, Cruel +3, Selfish +2  
+**Reputations:** Nizari Summoner 1
+
+**Combat:**  
+*Dodge*: Init +0, Attack -, Defense +4, Damage +0  
+*Dagger*: Init +0, Attack +6, Defense +3, Damage +1                                                                                                  
+
+**Soak:** +0  
+**Fatigue Leves:** OK, 0, -1, -3, -5, Unconscious  
+**Wound Penalties:** -1 (1-5), -3 (6-10), -5 (11-15), Incapacitated (16-20), Dead (21+)
+
+## Abilities:
+
++ Area Lore: Persia 2 (poisonous herbs)
++ Atheltics 2 (climbing)
++ Awareness 3 (alertness)
++ Bargain 1 (with demons)
++ Brawl 2 (dagger)
++ Chirurgy 2 (bind wounds)
++ Guile 3 (elaborate lies)
++ Infernal Lore 2 (demons)
++ Intrigue 2 (plotting)
++ Language (native): Western Iranian 5 (Persian)
++ Leadership 1 (demons)
++ Magic Lore 1 (jinn)
++ Medicine 5 (poisons)
++ Organization Lore: Mulhidun 1 (rituals)
++ Organization Lore: Nizari 2 (personalities)
++ Penetration 3 (Commanding)
++ Profession: Assassin 2 (poisoning)
++ Second Sight 2 (invisible things)
++ Stealth 4 (hiding)
+
+## Arts:
+
++ Summoning 13
++ Ablating 0
++ Binding 0
++ Commanding 9
++ Physic 7 [1]
+ 
+**Equipment:** none  
+**Encumbrance:** 0 (0)
+
+## Naranjs Known:
+
++ *Summon Al-Wahdat* (Summoning 10/+15)
```

Comment:
Is it possible to add the reference for non custom Naranj ? (Such as FTp42 for Finis Terra p42)

Reply:
Yes, I can reference them.